### PR TITLE
Korean typo correction

### PR DESCRIPTION
**Pull request by Terrorboy**
~# Validation.php~
~- `유요` => `유효`~

# 4.0.0* => 4.0.4
- Language/ko/CLI.php
- Language/ko/Database.php
- Language/ko/Fabricator.php
- Language/ko/Fabricator.php
- Language/ko/Format.php
- Language/ko/HTTP.php
- Language/ko/Migrations.php
- ~Language/ko/Seed.php~ (*Delete*)
- Language/ko/Session.php
- Language/ko/Validation.php
- README.rst

### CODE REVIEW

**Comment by Terrorboy** [comment]
@paulbalandan 
Is this test my fault?

**Comment by totoprayogo1916** [comment]
read [CONTRIBTING](https://github.com/codeigniter4/translations/blob/develop/CONTRIBUTING.md)

**Comment by paulbalandan** [comment]
@Terrorboy I know your PR only deals with the typo but while you're here can you also add/remove the keys so that `ko` will be updated? Please read the contributing guide on how to deal with this.

**Comment by Terrorboy** [comment]
I am fixing this.
please wait a moment.

**Comment by Terrorboy** [comment]
@totoprayogo1916 @paulbalandan

Thanks to this, we were able to fix it. Thank you.

File: Language/ko/CLI.php [hunk 1]
```diff
@@ -12,9 +12,19 @@
...
+	'altCommandSingular'  => '이게 맞습니까?', // 'Did you mean this?',
+	'altCommandPlural'    => '이중 하나입니까?', // 'Did you mean one of these?',
    'helpUsage'       => '사용:', // 'Usage:',
    'helpDescription' => '설명:', // 'Description:',
    'helpOptions'     => '옵션:', // 'Options:',
    'helpArguments'   => '인수:', //'Arguments:',
    'invalidColor'    => '유효하지 않은 {0} 색상: {1}.', // CI4 기준 : {0}에는 'foreground' 또는 'background' 문자열이 들어감. // 'Invalid {0} color: {1}.',
+
+	// Generators
+	'generateClassName'   => 'class 이름', // 'Name of class',
```
Comment (paulbalandan) [inline Language/ko/CLI.php:24]:
```suggestion
	'generateClassName'   => '클래스 이름', // 'Name of class',
```

File: Language/ko/Migrations.php [hunk 2]
```diff
@@ -43,12 +41,17 @@
...
    'added'             => '실행 중: ', // 'Running: ',
-
+   
+	// Migrate Status
+   'namespace'         => '네임스페이스',
    'version'           => '버전', // 'Version',
    'filename'          => '파일명', // 'Filename',
+	'group'             => '그룹',
+   'on'                => '마이그레이션 대상: ', // 'Migrated On: ',
+	'batch'             => 'Batch',
```
Comment (paulbalandan) [inline Language/ko/Migrations.php:56]:
```suggestion
	'group'          => '그룹',
	'on'                => '마이그레이션 대상: ', // 'Migrated On: ',
	'batch'           => 'Batch',
```

File: Language/ko/Session.php [hunk 3]
```diff
@@ -16,4 +16,5 @@
    'writeProtectedSavePath' => 'Session: 설정된 세션 저장 경로 "{0}"의 경우, PHP 프로세스에 의해 쓰기를 할 수 없습니다.', // 'Session: Configured save path "{0}" is not writable by the PHP process.',
    'emptySavePath'          => 'Session: 저장 경로가 설정되어 있지 않습니다.', // 'Session: No save path configured.',
    'invalidSavePathFormat'  => 'Session: 유효하지 않는 Redis 세션 저장 경로 형식: {0}', // 'Session: Invalid Redis save path format: {0}',
+   'invalidSameSiteSetting' => '세션 : SameSite 설정은 None, Lax, Strict 또는 빈 문자열이어야합니다. 주어진 값 : {0}', // 'Session: The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}',
```
Comment (paulbalandan) [inline Language/ko/Session.php:19]:
```suggestion
   'invalidSameSiteSetting' => 'Session : SameSite 설정은 None, Lax, Strict 또는 빈 문자열이어야합니다. 주어진 값 : {0}', // 'Session: The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}',
```

File: README.rst [hunk 4]
```diff
@@ -43,7 +43,7 @@ fr        French                4.0.0*       amuratet           Amuratet?
 id        Indonesian            4.0.0*       ridho1991          Mutasim Ridlo
 it        Italian               4.0.0        luk3b8             Luca Rasia
 ja        Japanese              4.0.0        kohenji01          Shoji Ogura
-ko        Korean                4.0.0*       Timothy-Y-H-Lee    Timothy-Y-H-Lee
+ko        Korean                4.0.4        codeigniter-kr     codeigniter-kr
```
Comment (paulbalandan) [inline README.rst:46]:
```suggestion
ko        Korean                4.0.2        codeigniter-kr     codeigniter-kr
```

**Review by paulbalandan — CHANGES_REQUESTED** [review]
Few more fixes before I merge this.

File: README.rst (hunk 4 above)
Comment (Terrorboy) [reply README.rst:46]: I checked based on the 4.0.4 repository, is it correct that this is marked as 4.0.2?

Comment (paulbalandan) [reply README.rst:46]: Yes, this was released in this repo as v4.0.2

**Review by Terrorboy — COMMENTED** [review]
Apply the suggestion.

**Comment by paulbalandan** [comment]
Update for Korean in relation to #148 done by this PR.

@Terrorboy  thank you.

**Pull request merged**